packages/components/src/utils/config-values.js

```diff
@@ -4,13 +4,41 @@
 import { space } from '../ui/utils/space';
 import { COLORS } from './colors-values';
 
-const CONTROL_HEIGHT = '30px';
+const CONTROL_HEIGHT = '36px';
```

Comment:
This is used by other components as well, so it might be safer to use a specific height for the segmented control.

Reply:
Depends. No controls should be 30px height as it's incompatible with the 12px grid. Which other controls use this?

Reply:
Hm.. Okay, we can stick with it then. These values are being ported but I don't think they are polished to match the GB so this seems like the right direction to conform with the current design.

Reply:
It's close, but there are a few values here and there that will need tuning as we port things over, so they feel at home.